extensions/basic_auth/README.md

```diff
@@ -57,24 +57,28 @@ message PluginConfig {
 
 // BasicAuth defines restriction rules based on three elements.
 message BasicAuth {
+  // HTTP host that this basic auth rule applies on.
+  // If host is not provided, this rule is applied to all hosts.
+  string host = 1;
```

Comment:
Wondering if this should be repeated?
Ideally you want to match a virtual host which is normally a list.

Reply:
Think a bit more about this, we probably should make it match cluster name instead of host name, otherwise it would be very easy to bypass the check. wdyt?

Reply:
Updated this to a simplified version of Envoy domain matching, and host now can be provided as a list.